src/components/ui/PaymentRequestAmountFilter/PaymentRequestAmountFilter.tsx

```diff
@@ -0,0 +1,197 @@
+﻿import FilterButton from '../FilterButton/FilterButton';
```

Comment:
I'd remove PaymentRequest from the name. We'll reuse these for Accounts Payable.